tests/test_cp_als.py

```diff
@@ -206,3 +206,28 @@ def test_cp_als_sptensor_zeros(capsys):
     capsys.readouterr()
     assert pytest.approx(output3["fit"], 1) == 0
     assert output3["normresidual"] == 0
+
+
+@pytest.mark.indevelopment
+def test_cp_als_tensor_printitn(capsys, sample_tensor):
+    (data, T) = sample_tensor
```

Comment:
Style note (you are following what is in this file, but I haven't revisited it recently so it should be cleaned up eventually), there is no need for the parenthesis when returning tuples. We also don't use data anywhere.
```suggestion
    _, T = sample_tensor
```